seedfarmer/commands/_bootstrap_commands.py

```diff
@@ -0,0 +1,97 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License").
+#    You may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+import json
+import logging
+import os
+from typing import Any, Dict, List, Optional
+
+import yaml
+from jinja2 import Template
+
+from seedfarmer import CLI_ROOT
+from seedfarmer.utils import CfnSafeYamlLoader
+
+_logger: logging.Logger = logging.getLogger(__name__)
+
+
+def get_toolchain_template(
+    project_name: str,
+    principalARN: List[str],
+    permissionsBoundaryARN: Optional[str] = None,
+) -> Dict[Any, Any]:
+    with open((os.path.join(CLI_ROOT, "resources/toolchain_role.template")), "r") as f:
+        role = yaml.load(f, CfnSafeYamlLoader)
+    if principalARN:
+        role["Resources"]["ToolchainRole"]["Properties"]["AssumeRolePolicyDocument"]["Statement"][0]["Principal"][
+            "AWS"
+        ] = principalARN
```

Comment:
won't this only works if `principalARN` is a string? or can the `AWS` key be a list of Principals?

Reply:
it is a list of strings...so even if you pass in one via CLI, it will be a list of 1 (I will confirm though)

Reply:
`seedfarmer bootstrap toolchain exampleprojtest -t arn:aws:iam::616260033377:role/Admin -t arn:aws:iam::616260033377:role/AwsSecurityAudit --synth` generate a tempalte with the following:
```
      AssumeRolePolicyDocument:
        Statement:
        - Action: sts:AssumeRole
          Effect: Allow
          Principal:
            AWS:
            - arn:aws:iam::616260033377:role/Admin
            - arn:aws:iam::616260033377:role/AwsSecurityAudit
        Version: '2012-10-17'
```

It successfully deployed